### PR TITLE
switch confd to go modules from glide

### PR DESCRIPTION
**Pull request by rafaelvanoni**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -69,62 +69,66 @@ PACKAGE_NAME?=github.com/kelseyhightower/confd
...
 ###############################################################################
-build: bin/confd
+.PHONY: remote-deps
+remote-deps:
+	$(DOCKER_RUN) $(CALICO_BUILD) sh -c '\
+	go list all; \
+	mkdir -p vendor/golang.org/x/crypto; \
+	cp -r `go list -m -f "{{.Dir}}" golang.org/x/crypto`/openpgp vendor/golang.org/x/crypto/openpgp; \
+	chmod -R +w vendor/golang.org/x/crypto'
+
```
Comment (spikecurtis) [inline Makefile]: I don't think these remote deps are needed.  I removed this section and things seem to build fine --- why did you add them?

Comment (rafaelvanoni) [reply Makefile]: Probably leftover from another issue, let me commit the removal...

Comment (rafaelvanoni) [reply Makefile]: Fixed
